### ISSUE TITLE
Missing \ here.

### ISSUE THREAD

**Issue by gsubramani** [RethinkRobotics/intera_sdk#145]
https://github.com/RethinkRobotics/intera_sdk/blob/876d64a6ec23dfc6556d2f9e0ef398272bc73b4b/intera_examples/scripts/set_interaction_options.py#L149
https://github.com/RethinkRobotics/intera_sdk/blob/876d64a6ec23dfc6556d2f9e0ef398272bc73b4b/intera_examples/scripts/set_interaction_options.py#L150

 Everything on line 150 is ignored so even valid quaternions raise an errors.

**Comment by ShiweiWang** [comment]
Thank you for pointing this out! We will address this in 5.3 release.

**Comment by ShiweiWang** [comment]
It turns out this was fixed in : https://github.com/RethinkRobotics/intera_sdk/pull/139
Please see the updated code in here: 
https://github.com/RethinkRobotics/intera_sdk/blob/release-5.3.0/intera_interface/src/intera_motion_interface/interaction_options.py#L186-L189
```
        # check for unit quaternion
        quat_sum_square = (interaction_frame.orientation.w * interaction_frame.orientation.w
                          + interaction_frame.orientation.x * interaction_frame.orientation.x
                          + interaction_frame.orientation.y * interaction_frame.orientation.y
                          + interaction_frame.orientation.z * interaction_frame.orientation.z)
```
This will be merged into `master` branch once 5.3 is released.
